SPECS/sympa-6.2.spec.in

```diff
@@ -530,6 +534,17 @@ getent passwd sympa >/dev/null || \
       -c "System User for Sympa" \
       -s "/sbin/nologin" \
       sympa
+# Fix CSS and pictures paths
+if [ $1 -gt 1 ]; then
+    if [ -d %{_datadir}/%{name}/static_content/css -o -d %{_datadir}/%{name}/static_content/pictures ]; then
+        cp -pur %{_datadir}/%{name}/static_content/{css,pictures} %{_localstatedir}/lib/%{name}/
```

Comment:
Is this better?
```
cp -pur %{_localstatedir}/%{name}/static_content/{css,pictures} %{_localstatedir}/lib/%{name}/
```

Reply:
Much better, as what you propose is correct, while what's in the commit is wrong. Thanks for catching this.
I've fixed the commit and force-pushed.